include/kpro_error_codes.hrl

```diff
@@ -0,0 +1,153 @@
+%% Generated code, do not edit!
```

Comment:
Why check in a generated file?

Reply:
Code generation chain in this repo looks like this:
From Java Classes ->  [priv/api-keys.eterm, priv/kafka.bnf] -> 
[include/kpro_error_codes.hrl, src/kpro_schema.erl].

Generated files are checked in mainly to simplify build scripts etc. (Makefile, rebar, rebar3).
Mostly, we need to re-generate when upgrading to support newer version KAFkA, but that doesn't happen often.
And when changed, it's nice to have a diff for review.